docs/source/basics/overview.rst

```diff
@@ -0,0 +1,191 @@
+Using strax
+================
+
+Strax is centralized around a centralized class: the `Context`. Users interact with the context to process and request data.
+On this page, we will build up an example context from scratch and how to interact with this context.
+
+A user will always interact with the context, after which the context will figure out the required action to perform such as loading data from disk, or processing data newly.
+
+
+.. image:: context.svg
+
+To start a new context, initialize the class from strax:
+
+
+.. code-block:: python
+
+    >>> import strax
+
+    >>> context = strax.Context()
+
+At this point, we still need to add the plugins and storage (explained below) to use the context.
+
+Plugins
+-------------
+To a context several classes are registered that are responsible for processing/generating the data. In the example-figure above, we have three plugins:
+ - PulseProcessing
+ - PeakProcessing
+ - EventProcessing
+
+Let's start by adding the first plugin (``PulseProcessing``):
+.. code-block:: python
```

Comment:
This is somehow not recognized at least it is written in the html too.

Reply:
Thanks a lot, let me fix it!